doc/openocd.texi

```diff
@@ -11369,16 +11369,24 @@ OpenOCD. When off, they generate a breakpoint exception handled internally.
 The commands below can be used to prevent OpenOCD from using certain RISC-V trigger features.
 For example in cases when there are known issues in the target hardware.
 
-@deffn {Command} {riscv set_enable_eq_match_trigger} [on|off]
-When on (default), allow OpenOCD to use exact-match triggers in watchpoints.
-@end deffn
+@deffn {Command} {riscv set_enable_trigger_type} [(@option{eq}|@option{napot}|@option{ge_lt}|@option{all}) (@option{wp}|@option{none})]
+Control usage of triggers in wp by OpenOCD. All trigger types are enabled by default.
+This command affects only on wp which will be set after the command.
 
-@deffn {Command} {riscv set_enable_napot_trigger} [on|off]
-When on (default), allow OpenOCD to use NAPOT trigger in watchpoints.
-@end deffn
+The following options for choosing trigger types can be used:
+
+@itemize @minus
+@item @option{eq}: Equality match trigger
+@item @option{napot}: NAPOT trigger
+@item @option{ge_lt}: Chained pair of `greater-equal` and `less-than` triggers
+@end itemize
```

Comment:
Please, add also the `@option{all}` to the list.

Reply:
Addressed.